shared/js/background/tab-manager.js

```diff
@@ -29,9 +29,9 @@ class TabManager {
      * 1. When a new tab is opened. See onUpdated listener below
      * 2. When we get a new main_frame request
      */
-    create(tabData) {
+    create(tabData, components) {
```

Comment:
Passing these as a global as using import instead does make it work via utils.js

However: this makes the build system interpret tab.js as ESM and that requires a whole lot of refactoring to match the correct wiring for types all over the codebase.

Where Tab is currently `/* @param {require('path/tab.js')} */` to `/* @param {import {tab} from 'path/tab.js' } */`